source/sass/buyers-guide/views/about.scss

```diff
@@ -135,3 +135,9 @@
     }
   }
 }
+
+#pni-contact {
+  .short-content {
+    min-height: 50vh;
```

Comment:
should this just always be the case? Are there any pages where the min-height should be less than half the screen? @kristina

Reply:
This should always be the case but as Mavis said in slack it should be separate ticket. Don't want to push too much today with extra visitors coming in.

Reply:
absolutely, I'm all for follow-ups and landing this first.

Reply:
filed https://github.com/mozilla/foundation.mozilla.org/issues/2612 for followup